cmd/plugin/builder/cli.go

```diff
@@ -58,6 +60,8 @@ func newCompileCmd() *cobra.Command {
 	cmd.Flags().StringVar(&compileArgs.ArtifactsDir, "artifacts", compileArgs.ArtifactsDir, "Path to output artifacts")
 	cmd.Flags().StringVar(&compileArgs.GoPrivate, "goprivate", "", "Comma-separated list of glob patterns of module path prefixes to set as GOPRIVATE on build")
 
+	cmd.Deprecated = fmt.Sprintf("Use %q instead.", "tanzu builder plugin build")
```

Comment:
```suggestion
	cmd.Deprecated = fmt.Sprintf("use %q instead.", "tanzu builder plugin build")
```
because Cobra adds this string after a `,`:
```
Command "compile" is deprecated, Use "tanzu builder plugin build" instead.
```